Tests/LibWeb/Text/input/HTML/DOMParser-parseFromString-document-url.html

```diff
@@ -0,0 +1,17 @@
+<script src="../include.js"></script>
+<script>
+    test(() => {
+        const parser = new DOMParser();
+
+        for (const mimeType of ["text/html", "application/xhtml+xml"]) {
+            const doc = parser.parseFromString("<html><b>hello", "text/html");
```

Comment:
`mimeType` doesn't actually get passed to `paraeFromString`.


```suggestion
            const doc = parser.parseFromString("<html><b>hello", mimeType);
```

It should be closer to this (although I don't know if `<html><b>hello` successfully parses under XHTML).